client/my-sites/plans-features-main/wpcom-faq.jsx

```diff
@@ -71,9 +71,9 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			/>
 
 			<FAQItem
-				question={ translate( 'Can I upload my own theme?' ) }
+				question={ translate( 'Can I install my own theme?' ) }
 				answer={ translate(
-					"Yes! With the WordPress.com Business plan you can upload any theme you'd like." +
+					"Yes! With the WordPress.com Business plan you can install any theme you'd like." +
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Yes! With the WordPress.com Business plan you can upload any theme you'd like. All plans give you access to our {{a}}directory of free and premium themes{{/a}}. These are among the highest-quality WordPress themes, hand-picked and reviewed by our team.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).